app/controllers/guest_rooms_controller.rb

```diff
@@ -0,0 +1,32 @@
+class GuestRoomsController < ApplicationController
+
+  def create
+    guest_room = GuestRoom.new(guest_room_params)
+
+    if guest_room.save 
```

Comment:
Nice job with the `.save` and using strong params; this leaves you set up for sad paths eventually. 👍